CodeGeneration/Sources/generate-swiftsyntax/templates/swiftsyntax/SyntaxRewriterFile.swift

```diff
@@ -101,7 +101,7 @@ let syntaxRewriterFile = SourceFileSyntax(leadingTrivia: .docLineComment(generat
       }
       """)
     
-    for baseKind in SYNTAX_BASE_KINDS.filter { !["Syntax", "SyntaxCollection"].contains($0) } {
```

Comment:
There was a warning `warning: trailing closure in this context is confusable with the body of the statement; pass as a parenthesized argument to silence this warning`